docs/user/security/securing-kibana.asciidoc

```diff
@@ -5,79 +5,62 @@
 <titleabbrev>Configure security</titleabbrev>
 ++++
 
-{kib} users have to log in when {stack-security-features} are enabled on your
-cluster. You configure roles for your {kib} users to control what data those
-users can access.
+When you start {es} for the first time, {stack-security-features} are enabled on
+your cluster and TLS is configured automatically. The security configuration
+process generates a password for the `elastic` user and an enrollment token for
+{kib}.
+{ref}/configuring-stack-security.html[Start the {stack} with security enabled]
+and then enroll {kib} as part of the configuration process.
 
-Most requests made through {kib} to {es} are authenticated by using the
-credentials of the logged-in user. There are, however, a few internal requests
-that the {kib} server needs to make to the {es} cluster. For this reason, you
-must configure credentials for the {kib} server to use for those requests.
+You can then log in to {kib} as the `elastic` user to create additional roles
+and users. You can also configure <<elasticsearch-mutual-tls,mutual TLS>>
+between {kib} and {es} where requests are authenticated by both applications.
 
-With {security-features} enabled, if you load a {kib} dashboard that accesses
-data in an index that you are not authorized to view, you get an error that
-indicates the index does not exist. The {security-features} do not currently
-provide a way to control which users can load which dashboards.
+NOTE: With {security-features} enabled, if you load a {kib} dashboard that
+accesses data in an index that you are not authorized to view, you get an error
+that indicates the index does not exist. The {security-features} do not
+currently provide a way to control which users can load which dashboards.
```

Comment:
I know you didn't add this, but taking a hard look at this page, I don't really like this at all. It's sort of ... too specific and doesn't really add value to this page, and the last sentence is confusing.

If we feel that we need _something_, I'd suggest shortening it to something like:
"NOTE: When a user is unauthorized to view data in Kibana, such as an Elasticsearch index, it will appear to them as if the index does not exist."

But that still just feels out of place to me. Maybe Kaarina or Gail will have an opinion 🤷

Reply:
I think that we should distinguish that if users cannot view data in an index, then the entire index will seem to not exist. "Seem to not exist" feels really ephemeral though, so I'd opt for something more concrete. Something like:

>When a user is not authorized to view data in an index (such as an Elasticsearch index), the entire index will be inaccessible and not display in Kibana.